### PR TITLE
Packages renaming & multiple fixes

### LINKED ISSUE #169: exe/server: use go-homedir
As @sanderpick  suggested, use `go-homedir` to make a cleaner default of default repopath.

Mentioned in #160

### LINKED ISSUE #170: index/slashing: save partial progress
Currently, the slashing index only saves information when finish processing from the last known chain height to the current head.

If that delta is long, that might take some time. Since work is processed in batches, we could save new information as batches finish so not needing to wait until all is finished to leverage new information.

### LINKED ISSUE #171: exe/server: use Go 1.14 in Dockerfile
Switch compiler to Go 1.14.

### LINKED ISSUE #173: readme fix based on cli update
this section is now out of date, 

https://github.com/textileio/powergate/#client-filcoin

### PR DESCRIPTION
**Pull request by jsign**
* Renames _FPA_ to _FFS_.
* Renames _FastAPI_ to _API_ with _api_ as shorthand.
* Rename repo to _Powergate_
   * Repo name changed from _.texfc_ to _.powergate_
   * Binaries changed from _fil-tools(d)_ to _pow(d)_
   * Env var prefix changed from _TEXFILTOOLS__ to _TEXPOWERGATE__
* Closes #169
* Closes #170
* Closes #171
* Closes #173

### CODE REVIEW

**Comment by asutula** [comment]
There are some actual bug fixes and/or improvements in here, right? Can you point those out with a few comments just to make sure they don't get lost in the naming refactor changes?

File: exe/server/main.go [hunk 1]
```diff
@@ -65,16 +65,16 @@ func main() {
...
+		if repoPath == "~/.powergate" {
+			expandedPath, err := homedir.Expand(repoPath)
 			if err != nil {
-				log.Fatalf("error when setting default repo path to home dir: %s", err)
+				log.Fatalf("expanding homedir: %s", err)
 			}
-			repoPath = strings.Replace(repoPath, "${HOME}", home, -1)
+			repoPath = expandedPath
 		}
 	} else {
```
Comment (jsign) [inline exe/server/main.go]: Use of `go-homedir`

File: index/slashing/slashing.go [hunk 2]
```diff
@@ -193,17 +193,16 @@ func (s *SlashingIndex) updateIndex() error {
 			return err
 		}
 		log.Infof("processed from %d to %d", path[i].Height(), path[j-1].Height())
+		s.lock.Lock()
+		s.index = index
+		s.lock.Unlock()
```
Comment (jsign) [inline index/slashing/slashing.go]: Save partial progress.

File: exe/server/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.13.7 as builder
+FROM golang:1.14 as builder
```
Comment (jsign) [inline exe/server/Dockerfile]: Bump to 1.14

**Comment by jsign** [comment]
@asutula , I left some extra comments about the fixes.
I'll merge just to unblock @andrewxhill , but feel free to take a look and I'll address if something pops up.

**Pull request merged**